management/utils.py

```diff
@@ -43,6 +43,20 @@ def load_settings(env):
     except:
         return { }
 
+# THE SSH KEYS AT /root/.ssh
+
+def load_ssh_public_key():
+    ssh_public_key_file = os.path.join('/root', '.ssh', 'id_rsa_miab.pub')
+    if os.path.exists(ssh_public_key_file):
+        with open(ssh_public_key_file, 'r') as f:
+            return f.read()
+
+def load_ssh_private_key():
```

Comment:
I don't think this function should be included since it is not used anywhere

Reply:
Solved with d1f21d601ecec10f591041e716bbd36552333cc9.